gbpservice/neutron/nsf/config_agent/RestClientOverUnix.py

```diff
@@ -0,0 +1,112 @@
+import os
+import sys
+import httplib2
+import httplib
+import six
+import six.moves.urllib.parse as urlparse
+import socket
+import exceptions
+
+
+class UnixDomainHTTPConnection(httplib.HTTPConnection):
+
+    """Connection class for HTTP over UNIX domain socket."""
+
+    def __init__(self, host, port=None, strict=None, timeout=None,
+                 proxy_info=None):
+        httplib.HTTPConnection.__init__(self, host, port, strict)
+        self.timeout = timeout
+        self.socket_path = '/tmp/uds_socket'
+
+    def connect(self):
+        self.sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
```

Comment:
How are errors for any of the socket calls handled? Is there try...except block to capture these errors?